static/scss/partials/dashboard.scss

```diff
@@ -209,9 +213,35 @@
     }
 
     input {
+        color: $color-grey-50;
+        display: flex;
+        font-weight: 500;
+        align-items: center;
+        background: transparent;
+        font-size: 1em;
+        padding: 6px $spacing-lg 6px 6px;
```

Comment:
Could we use protocol tokens here?

Reply:
I left it as `6px` because I had to choose between `4px` or `8px`. If I had to pick, which do you suggest going for?

Reply:
`$spacing-md` will be fine! 😄